src/views/mfa-verify/PasswordForm.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2016-2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['okta', './inputs/ShowPasswordBox'], function (Okta, PasswordBox) {
+
+  var _ = Okta._;
+
+  return Okta.Form.extend({
+    className: 'mfa-verify-question',
```

Comment:
should this be ```mfa-verify-password```? and maybe we need to update some css class as well

Reply:
👍 Missed that. Thanks!